src/_data/whats-new.yml

```diff
@@ -4,8 +4,208 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Mar 30 08:59:14 2020
+updated: Wed Apr 22 10:36:00 2020
 entries:
+- description: Added 'Set a fixed price for bundled products' to the [REST usage notes](https://devdocs.magento.com/guides/v2.3/rest/notes.html)
+    topic.
+  versions: 2.3.x
+  type: Major Update
+  date: April 20, 2020
+  link: https://github.com/magento/devdocs/pull/7035
+- description: Added correct db schema information in the [Configure declarative schema](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.html)
+    topic.
+  versions: 2.3.x
+  type: Technical
+  date: April 16, 2020
+  link: https://github.com/magento/devdocs/pull/7045
+- description: Added a new reference topic for the [system.xml](https://devdocs.magento.com/guides/v2.3/config-guide/prod/config-reference-systemxml.html)
+    configuration file.
+  versions: 2.3.x
+  type: New Topic
+  date: April 16, 2020
+  link: https://github.com/magento/devdocs/pull/7024
+- description: 'New Page Builder topic on how to configure three different options
+    for styling elements in your content types: [How to configure styling options](https://devdocs.magento.com/page-builder/docs/how-to/how-to-configure-styling-options.html).'
+  versions: 2.3.x
+  type: Technical
```

Comment:
```suggestion
  type: New Topic
```

Reply:
@bdenham I wrote a huge comment above about why this topic should not be edited manually 🤦

Reply:
I realize that. And I understand the reason for it. I also fixed my original PR to match the change I suggested. 

![image](https://user-images.githubusercontent.com/1828494/80141326-fdd96100-856e-11ea-9828-fd58b221db9e.png)

So I have essentially aligned the PR so that the file CAN be manually edited, in this case. I understood your post. But I also provided the option to expedite the fix in this instance. 

I do not appreciate the hand over face emoji.